application/src/main/java/org/opentripplanner/updater/vehicle_rental/datasources/params/AllowedRentalType.java

```diff
@@ -0,0 +1,32 @@
+package org.opentripplanner.updater.vehicle_rental.datasources.params;
+
+import org.opentripplanner.framework.doc.DocumentedEnum;
+
+/**
+ * This is temporary and will be removed in a future version of OTP.
+ *
+ * Enum to specify the type of rental data that is allowed to be read from the data source.
+ */
+public enum AllowedRentalType implements DocumentedEnum<AllowedRentalType> {
```

Comment:
@optionsome / @leonardehrenfried  What is the right name for this?

I would not include "Allowed" in the name, it restrict the type, so it can not be reused.

Reply:
We discussed that we want to use a Set of opt-in rental types. I've used the same pattern when filtering what to import from NeTEx: https://github.com/opentripplanner/OpenTripPlanner/blob/f4bfedfc41d3b92df45927a72edcab28624510d6/application/src/main/java/org/opentripplanner/netex/loader/parser/NetexDocumentParser.java#L82-L84

In our case we probably want to use opt-in rather than opt-out as it's easier to reason about.